modules/olmv1-updating-an-operator.adoc

```diff
@@ -187,9 +194,9 @@ metadata:
 spec:
   packageName: openshift-pipelines-operator-rh
   installNamespace: <namespace>
-  version: "1.12.1" <1>
+  version: "1.15.0" <1>
```

Comment:
@michaelryanpeter the example is not correct. it still use old format, for example 
olm.operatorframework.io/v1alpha1
```yaml
spec:
  packageName: openshift-pipelines-operator-rh
  installNamespace: <namespace>
```